proto/src/determined/api/v1/api.proto

```diff
@@ -588,6 +588,17 @@ service Determined {
       tags: "Internal"
     };
   }
+  // Gather a trial's rendezvous info. Blocks until all trial containers connect
+  // to gather their rendezvous information and responds to them all at once.
+  rpc GetTrialRendezvousInfo(GetTrialRendezvousInfoRequest)
+      returns (GetTrialRendezvousInfoResponse) {
+    option (google.api.http) = {
+      get: "/api/v1/trials/{trial_id}/rendezvous_info/{container_id}"
```

Comment:
wdyt @rb-determined-ai ? this or just make it a POST?

Reply:
POST seems more correct to me, since it is not cacheable